### PR TITLE
add : added navigation and modal component

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
1.  Added navigation and modal component 
2.  Added documentation of  navigation and modal component

snap shot of navigation component 
image

snap shot of Modal component 
image

### CODE REVIEW

File: components/nav-modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        <div class="text pb2 " id="modal">
+            <h2 class="pb1 h2 comp-title">Modal</h2><br>
+            <p class="pb2">Add dialogs to your site for lightboxes, user notifications, or completely custom content.
+            </p>
+            <h3>Example</h3>
+        </div>
+
+        <div class="modal-container">
+            <div class="modals modal-row1">
+                <h3>Modal Title</h3><button class="modal-close">X</button>
```
Comment (gauravsinhaweb) [inline components/nav-modal/modal.html]: Use icon instead of letter x

File: components/nav-modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+.modal-close:hover {
+    color: var(--black);
+}
+
+.modal-row2 {
+    font-size: 16px;
+}
+
+.modal-row3 button {
+    font-size: 18px;
```
Comment (gauravsinhaweb) [inline components/nav-modal/modal.css]: Change px to rem for good responsiveness.

File: components/nav-modal/modal.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+.modal-row1 {
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 1px solid var(--grey);
+}
+
+.modal-row3 {
+    display: flex;
+    justify-content: right;
```
Comment (gauravsinhaweb) [inline components/nav-modal/modal.css]: Justify content: right Is not a css property.

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: components/nav-modal/nav.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+.nav-links-C a:hover,
+.nav-links-C a:active {
+    background-color: var(--blue);
+    padding: 0.62rem;
+}
+
+
+@media screen and (max-width:1000px) {
+    .logo-C {
+        font-size: 27px;
```
Comment (Muzammil-khan710) [inline components/nav-modal/nav.css]: Try rem here

File: components/nav-modal/nav.css [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    top: 0;
+}
+
+.navbar-comp ul {
+    float: right;
+    margin-right: 2rem;
+}
+
+.logo-C {
+    font-size: 34px;
```
Comment (Muzammil-khan710) [inline components/nav-modal/nav.css]: Try rem ?

Comment (JuvitaSaini09) [reply components/nav-modal/nav.css]: I have updated unit,thanks for the review

File: components/nav-modal/nav.css (hunk 4 above)
Comment (JuvitaSaini09) [reply components/nav-modal/nav.css]: I have upadated unit,thanks for the review

File: components/nav-modal/modal.css (hunk 3 above)
Comment (JuvitaSaini09) [reply components/nav-modal/modal.css]: I have fixed it, thanks for the review

File: components/nav-modal/modal.css (hunk 2 above)
Comment (JuvitaSaini09) [reply components/nav-modal/modal.css]: I have made the change,thanks for the review

File: components/nav-modal/modal.html (hunk 1 above)
Comment (JuvitaSaini09) [reply components/nav-modal/modal.html]: I have updated it, thanks for review

**Review by Muzammil-khan710 — APPROVED** [review]
Suggestion: You have defined colors in multiple CSS pages try creating one file for colors and import that file link to your all components*. Maintain good spacing in your code.. somewhere you have given spaces and somewhere not.. 

Approving as of now.

After all it is good to merge !

**Review by gauravsinhaweb — APPROVED** [review]

**Pull request merged**